lib/Service/RecipeService.php

```diff
@@ -137,13 +137,17 @@ public function checkRecipe($json) {
 
         // Make sure that "keywords" is an array of unique strings
         if(isset($json['keywords']) && is_string($json['keywords'])) {
-            $keywords = $json['keywords'];
-            $keywords = strip_tags($keywords);
-            $keywords = str_replace(',,', ',', $keywords);
-            $keywords = explode(',', $keywords);
-            $keywords = array_unique($keywords);
-            $keywords = implode(',', $keywords);
-            $json['keywords'] = $keywords;
+          $keywords = trim($json['keywords']);
+          $keywords = trim($keywords, ',');
+          $keywords = trim($keywords)
```

Comment:
This could have been simplified into one line by `trim($json['keywords'], " \t\n\r\o\x0B,");`. But it does not hurt like this.

Reply:
I actually tried that, but I think I see what I did wrong. I think I used single-quotes instead of double. Sigh.

Reply:
Yes, then PHP does interpret fewer symbols in a special way. Similar to variable expansion in double quoted strings while not in single quoted strings.